vimode/src/cmds/motion.c

```diff
@@ -40,42 +40,53 @@ void cmd_goto_right(CmdContext *c, CmdParams *p)
 	SET_POS(p->sci, pos, TRUE);
 }
 
+static gint doc_line_from_visible_delta(CmdParams *p, gint line, gint delta, gint *previous)
+{
+	gint step = delta < 0 ? -1 : 1;
+	gint new_line = p->line;
```

Comment:
Should be `line` instead of `p->line` - we don't always pass `p->line` as the parameter of this function.